messytables/jsontableschema.py

```diff
@@ -0,0 +1,115 @@
+import json
+
+unicode = type(u'')
+
+class FormatError(Exception): pass
+class DuplicateFieldId(Exception): pass
+
+class JSONTableSchema(object):
+    """
+    A library to handle JSON Table Schema files.
+    See http://www.dataprotocols.org/en/latest/json-table-schema.html
+    Original source: https://pypi.python.org/pypi/json-table-schema
+    This remains licenced under the Apache Licence v2.0
+    Original by Martin Keegan - martin.keegan@okfn.org
+    Changed by David McKee - dave.mckee@gmail.com
+    Modified to make Python 3 compatible
```

Comment:
In discussion with the author. Yes; it would be; will check if theirs is now Py3 compatible.